Readme.md

```diff
@@ -7,4 +7,23 @@ Current APIs:
 
 - S3
 
-Targeted for Java 1.6+
\ No newline at end of file
+Targeted for Java 1.6+
+
+**Currently, read timeout is disabled to avoid conflicts in updating.**
+
+Here is an example for a conflict occurred with only one user when read timeout is enabled:
```

Comment:
instead of spelling this out here in the readme, leave the note about disabling read timeout, but put a link to the wiki page that says "for more information about timeouts, please see the following wiki page:"